README.md

```diff
@@ -1,7 +1,11 @@
 InfraKit
 ========
-
+<!--
```

Comment:
?

Reply:
The CircleCI badge all of a sudden stops working and I can't figure out why it's stuck at "failing" even though master builds successfully.  Any ideas?  I am not thrilled with CircleCI anymore.  Time to look at other options.